ros_ign_point_cloud/src/point_cloud.cc

```diff
@@ -33,13 +33,13 @@
 #include <sensor_msgs/fill_image.h>
 #include <sensor_msgs/point_cloud2_iterator.h>
 
-IGNITION_ADD_PLUGIN(
-    ros_ign_point_cloud::PointCloud,
+GZ_ADD_PLUGIN(
```

Comment:
This breaks Fortress support - you won't notice on CI because this package isn't being built

Reply:
Ah.. Reverting

Reply:
https://github.com/gazebosim/ros_gz/pull/280/commits/1451eb70bc7bd8d8f9cb02eef54ba93fdf2d57f7